examples/validate/check_cpu_memory.yaml

```diff
@@ -35,4 +35,4 @@ spec:
               - name: "*"
                 resources:
                   requests:
-                    cpu: "<4"
+                    cpu: "<4m"
```

Comment:
Will resource be allowed with value "2Mi"? If not, I suggest to put wildcard here "<2m*"